diff -urN thunar-thumbnailers-0.0.1svn-r02578/configure.in thunar-thumbnailers-0.0.1svn-r02578.new/configure.in
--- thunar-thumbnailers-0.0.1svn-r02578/configure.in	2007-03-30 13:44:17.000000000 +0200
+++ thunar-thumbnailers-0.0.1svn-r02578.new/configure.in	2007-04-05 21:59:29.000000000 +0200
@@ -39,61 +39,29 @@
 dnl *** Check for basic programs ***
 dnl ********************************
 AC_PROG_INSTALL()
-AC_PATH_PROG(CONVERT, convert)
-      if test -z "$CONVERT"; then
-	 AC_MSG_ERROR([ImageMagick is required to build thunar-thumbnailers])
-      fi
 
 dnl *** check if we want raw image thumbnailing ***
 AC_ARG_ENABLE(enable-raw,
    [ --enable-raw Enables thumbnailing of the Raw images used by some digital cameras (requires dcraw). ],
    enable_raw=yes)
-if test "$enable_raw" = "yes"; then
-    dnl *** check to see if dcraw, needed for raw image thumbnailing, is on the path ***
-    AC_PATH_PROG(DCRAW, dcraw)
-	if test -z "$DCRAW"; then
-	    AC_MSG_ERROR([Install dcraw before enabling raw image thumbnailing])
-	fi
-fi
 AM_CONDITIONAL(ENABLE_RAW, test "$enable_raw" = "yes")
 
 dnl *** check if we want latex thumbnailing ***
 AC_ARG_ENABLE(enable-tex,
    [ --enable-tex Enables thumbnailing of latex and tex files (requires tetex). ],
    enable_tex=yes)
-if test "$enable_tex" = "yes"; then
-    dnl *** check to see if latex, needed for tex thumbnailing, is on the path ***
-    AC_PATH_PROG(LATEX, latex)
-	if test -z "$LATEX"; then
-	    AC_MSG_ERROR([Install tetex before enabling latex thumbnailing])
-	fi
-fi
 AM_CONDITIONAL(ENABLE_TEX, test "$enable_tex" = "yes")
 
 dnl *** check if we want grace (agr) thumbnailing ***
 AC_ARG_ENABLE(enable-grace,
    [ --enable-grace Enables thumbnailing of xmgrace / grace (.agr) files (requires grace). ],
    enable_grace=yes)
-if test "$enable_grace" = "yes"; then
-    dnl *** check to see if gracebat, needed for tex thumbnailing, is on the path ***
-    AC_PATH_PROG(GRACE, gracebat)
-	if test -z "$GRACE"; then
-	    AC_MSG_ERROR([Install grace before enabling grace/xmgrace thumbnailing])
-	fi
-fi
 AM_CONDITIONAL(ENABLE_GRACE, test "$enable_grace" = "yes")
 
 dnl *** check if we want video (ffmpeg) thumbnailing ***
 AC_ARG_ENABLE(enable-ffmpeg,
    [ --enable-ffmpeg Enables thumbnailing of video files supported by ffmpeg (requires ffmpegthumbnailer). ],
    enable_ffmpeg=yes)
-if test "$enable_ffmpeg" = "yes"; then
-    dnl *** check to see if ffmpegthumbnailer, needed for ffmpeg thumbnailing, is on the path ***
-    AC_PATH_PROG(FFMPEG, ffmpegthumbnailer)
-	if test -z "$FFMPEG"; then
-	    AC_MSG_ERROR([Install ffmpegthumbnailer before enabling ffmpeg-video thumbnailing])
-	fi
-fi
 AM_CONDITIONAL(ENABLE_FFMPEG, test "$enable_ffmpeg" = "yes")
 
 dnl *** check if we want to run update-mime-database on installed mimetypes ***
